workflow-core/src/main/java/com/squareup/workflow/LifecycleWorker.kt

```diff
@@ -30,6 +31,8 @@ import kotlinx.coroutines.suspendCancellableCoroutine
  */
 abstract class LifecycleWorker : Worker<Nothing> {
```

Comment:
Will deprecate this in a follow-up PR, since it's better implemented by a plain side effect now.

Reply:
#114